lib/gobstones/gbb_parser.rb

```diff
@@ -0,0 +1,59 @@
+require_relative 'board'
+require_relative 'cell'
```

Comment:
Since you have little support from ide and adding requires to each file is a bit tedious, it is a good practice to do not use requires within the same module - aka namespace - but to require all their files in just a main file that has the module definition. 

Then code outside the module will require only that main file